src/components/learner-credit-management/cards/Collapsibles.jsx

```diff
@@ -20,10 +20,6 @@ export const NextStepsForAssignedLearners = ({ course }) => (
           is calculated based on the course enrollment deadline or {ASSIGNMENT_ENROLLMENT_DEADLINE} days
           past the date of assignment, whichever is sooner.
         </li>
-        <li>
-          Learners will receive automated reminder emails every 10-15 days until the enrollment
-          deadline is reached.
-        </li>
```

Comment:
Removed based on this recent Figma annotation:

<img width="286" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/fc0d6314-92c8-47c0-9db6-8684948fc158">